### DISCUSSION TITLE
Cannot find java-udf jar on branch-1.2-lts after compiling

### DISCUSSION THREAD

**Discussion by m3ngzhang** [apache/doris#16339] [A - General / Q&A]
I did "sh build.sh -j 8" as the instruction told. I cannot find java-udf-jar under output/be/lib which is supposed to be there. Need some help.

**Comment by LemonLiTree** [comment]
Is the compilation completed normally？

**Reply by m3ngzhang** [reply]
> Is the compilation completed normally？

yes

**Reply by LemonLiTree** [reply]
I have this jar package after compiling today

**Reply by m3ngzhang** [reply]
> I have this jar package after compiling today

I pulled and tried again. Finally found it. It seems someone fixed it. Thx for replay.
